project/cmake/scripts/common/Macros.cmake

```diff
@@ -509,13 +494,14 @@ endfunction()
 
 # Parses git info and sets variables used to identify the build
 #
-# The following variables are set:
+# The following variables may be set:
 #   APP_SCMID - git HEAD commit in the form of 'YYYYMMDD-hash'
 #               if git tree is dirty, value is set in the form of 'YYYYMMDD-hash-dirty'
 #               if no git tree is found, value is set in the form of 'YYYYMMDD-nogitfound'
 #   GIT_HASH  - git HEAD commit in the form of 'hash'. if git tree is dirty,
 #               value is set in the form of 'hash-dirty'
-function(core_find_git_rev)
+#               if no git tree is found, value is set in the form of 'nogitfound'
+function(core_find_git_rev stamp)
```

Comment:
Thx! Just wondering if we should aligning this a bit with cmake convention? the function could take the output variable name as first parameter (as it does already, but don't restrict the names to those 2) and have an [option](https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html#command:cmake_parse_arguments) that defines if the date is prefixed?
If we ever want to store the version in a different variable for some reason, with the current function it wouldn't work.